### ISSUE TITLE
[Not ready for Triage/Review Meeting][Win32][Accessibility] [A11yMAS][Screen Readers][MAS4.2.12][ Message displayed under console is not announced by the screen reader when user press enter on Run button.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#727]
**User Experience:**
Screen reader user will not get the information of the value present in the field if screen reader is not announcing the value.

**Test Environment:**
OS Build: 16299.19
Excel version:1712(8808.1000)
Tools Used: Narrator
Prerequisites:
Add script lab “Add ON” In Excel

**Attachments:**
[MAS4.2.12_Notification is not announced.zip](https://github.com/OfficeDev/script-lab/files/1473680/MAS4.2.12_Notification.is.not.announced.zip)

**Repro Steps:**
1.	Open Script Lab in Excel
2.	Navigate to Run button and activate it.
3.	Navigate to Run pane Using F6 key
4.	Navigate to “Run” button  and activate it.
5.	Verify whether displayed message is announced by screen reader.

**Current Behavior:**
Message displayed under console is not announced by the screen reader when user press enter on Run button.

**Expected Behavior:**
Screen reader should announce the message so that user will get to know the changes happened at which location in the excel.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.2.12
